File: js/sumOfTwoSmallestPositiveNumbers.js
// Create a function that takes an array of numbers and returns the sum of the two lowest positive numbers.
// https://edabit.com/challenge/GNgCfKHWfQwByBNqa

const sumTwoSmallestNums = (arr) =>
  arr
    .filter((num) => num > 0)
    .sort((a, b) => a - b)
    .slice(0, 2)
    .reduce((a, b) => a + b, 0);

module.exports = sumTwoSmallestNums;

File: CSharp/SumOfTwoSmallestPositiveNumbers.cs
using System.Linq;

namespace CSharp
{
    // Create a function that takes an array of numbers and returns the sum of the two lowest positive numbers.
    // https://edabit.com/challenge/7F8ZhHpxeW7K65XRL
    public static class SumOfTwoSmallestPositiveNumbers
    {
        public static int SumSmallest(int[] values) => values.Where(num => num > 0).OrderBy(num => num).Take(2).Sum();
    }
}